doc/ext/sphinx_gallery/docs_resolv.py

```diff
@@ -0,0 +1,435 @@
+# -*- coding: utf-8 -*-
+# Author: Óscar Nájera
```

Comment:
@emmanuelle is there a reference to the original source code somewhere in this PR? It might be useful in the future in case of bug fixes/enhancements...?